dashboard/src/test/kotlin/com/ritense/dashboard/service/DashboardDataServiceIntTest.kt

```diff
@@ -71,6 +81,10 @@ class DashboardDataServiceIntTest @Autowired constructor(
     }
     @Test
     fun `should get data from test datasource`() {
+        runWithoutAuthorization {
+            dashboardService.createDashboard("Test", "Test description")
```

Comment:
I guess the point of this is to prove that the just created dashboard is not returned (or is), but it is not explicitly checked that this one is not included.

Reply:
Not fixing this one, I think this is okay but could be improved in the future.

Reply:
That future will never happen, and you know it.